doc/examples/scraper/README.md

```diff
@@ -1,6 +1,6 @@
 # Quick Start Guide: Web Scraper
 In this guide you're going to create a Pachyderm pipeline to scrape web pages. 
-We'll use a standard unix tool, `wget`, to do our scraping. 
+We'll use a standard unix tool, `wget`, to do our scraping.
```

Comment:
I agree that these spurious diff lines hurt readability and that we should have company policy for this stuff. The easiest solution here was #924, and once that's submitted these spurious diff lines will go away.